docs/crio.8.md

```diff
@@ -365,7 +365,7 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--pause-command**="": Path to the pause executable in the pause image. (default: "/pause")
 
-**--pause-image**="": Image which contains the pause executable. (default: "registry.k8s.io/pause:3.9")
+**--pause-image**="": Image which contains the pause executable. (default: "registry.k8s.io/pause:3.10")
```

Comment:
We should add the pause image to [dependencies.yaml](https://github.com/cri-o/cri-o/blob/main/dependencies.yaml). Do you want to take care of that now or should we follow-up on it?

Reply:
@tukwila, will you be able to update dependencies too?

We can then go ahead and merge these changes.